html-and-the-web/html-and-the-web_div-salad/index.html

```diff
@@ -13,59 +13,59 @@
   </head>
   <!--someone forgot to wrap the content of our webpage with an HTML element,
 	which one was it again?-->
-  <div>
-    <div>
-      <div>Jane Doe</div>
+  <body>
+    <header>
+      <h1>Jane Doe</h1>
       <!--Which elements could be used to link to parts of the webpage,
 		which appear below?-->
-      <div id="nav">
-        <div>About Me</div>
-        <div>Contact</div>
-        <div>Skillset</div>
-      </div>
-    </div>
+      <nav id="nav">
+        <a href="./about.html">About Me</a>
+        <a href="./contact.html">Contact</a>
+        <a href="./skillset.html">Skillset</a>
+      </nav>
+    </header>
     <!--Looks like the following part is the central content of our webpage.-->
-    <div>
-      <div id="about-me">
-        <div>About me</div>
-        <div>
+    <main>
+      <section id="about-me">
+        <h2>About me</h2>
+        <p>
           Hi there! Welcome to my personal website. Here is gathered the most
           important information about me. If you want to work with me or get to
           know me, please don't hesitate to get in contact.
-        </div>
+        </p>
         <!--What element could be used to combine an image with a descriptive caption?-->
-        <div>
+        <figure>
           <img
             src="https://source.unsplash.com/random/400x300?fruit"
             alt="a random picture"
           />
-          <div>
+          <figcaption>
             A random picture, because I would like to maintain my privacy and
             let my code speak for me, instead of images.
-          </div>
-        </div>
-      </div>
+          </figcaption>
+        </figure>
+      </section>
 
-      <div id="contact">
-        <div>Contact</div>
+      <section id="contact">
+        <h2>Contact</h2>
         <!--When clicking on these element, nothing happens! But I want to be redirected to the other websites. Why is it not working?-->
         <div href="https://github.com/spiced-academy">GitHub</div>
```

Comment:
I think, it is better to use <a href... instead of <div href... in these 3 lines.

Reply:
arrrggghhh I thought I done that....Thanks Utku, I made the changes. Please approve when you have time.

Reply:
@Scot-MacDonald all seems fine to me now, you can merge it :)